src/wrapper/data_chunk.ml

```diff
@@ -28,6 +28,30 @@ let get_opt t type_ idx =
 ;;
 
 let free t ~here = Resource.free t.data_chunk ~here
+let column_count count _t = count
```

Comment:
I don't think we want to have a function like this which doesn't do anything; delete?

Reply:
I don't think you've addressed this issue.